cicd/gitops/setting-up-argocd-instance.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="setting-up-argocd-instance"]
+= Setting up a new Argo CD instance
+include::_attributes/common-attributes.adoc[]
+:context: setting-up-argocd-instance
+
+toc::[]
+
+By default, the {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources.
+
+To manage cluster configurations or deploy applications, you can install and deploy a new Argo CD instance. By default, any new instance has permissions to manage resources only in the namespace where it is deployed.
+
```

Comment:
Make these two sentences one paragraph, then it is more closely aligned to the upcoming Jupiter requirements for abstracts:
https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-ContentChecklist